### ISSUE TITLE
wrong flaps logic - FLAPS 1 / 1+F

### ISSUE THREAD

**Issue by andrejetcat** [flybywiresim/a32nx#4011]
**Mod Version**

ccf27c4  Dev+FBW

**Describe the bug**

select "flaps 1" in the approach goes into "flaps 1+F"
Flaps 1+F is only used for takeoff, while in this position slats AND flaps are out. for Landing (approach) flaps 1 position only extend the slats. Only flaps 2 position activate the flaps and so on.
--------
![flaps bug](https://user-images.githubusercontent.com/72918017/111478002-f47d5800-872f-11eb-8dbb-d4dc6c83e216.jpg)

When i select flaps 1 for takeoff, both (slats and flaps) start to extend correctly, like IRL.
But on Approach, when I select "flaps 1" @ first, only slats move like it should. BUT only AFTER slats extension, the flaps starts to move in position 1.

**To Reproduce**

1.cold dark start
2.approch to an airport of your choise
3.select "flaps 1" for landing - watch ECAM 

**Expected behavior**

Flaps 1+F is only used for takeoff, while in this position slats AND flaps are out. for Landing (approach) flaps 1 position only extend the slats. Only flaps 2 position activate the flaps and so on.

**Actual behavior**

approach" Flaps 1" leads to "flaps 1+F" Position

**References**

**Additional context**

Was this working before/when did the issue start occurring?
don´t know
Is this a problem in the vanilla unmodded game?
don´t know

Discord username (if different from GitHub):

**Comment by MisterChocker** [comment]
The version you referring too is quite old, can you please update and retry?

**Comment by andrejetcat** [comment]
![installer](https://user-images.githubusercontent.com/72918017/111483433-10cfc380-8735-11eb-9ef3-55350de3ddad.jpg)

The installer said, that is the up-to-date-version. Thats strange. 
I will load the version from this side and retry.

**Comment by andrejetcat** [comment]
so, now when i´m loading the Dev+FBW from this side, the problem with the ND´s remains INOP is back. 
strange!!!

**Comment by andrejetcat** [comment]
...sorry for this amount of posts from my side. I try it again. Same version, NO changes. One time I start the sim, ND´s do not work. SEcound time it start the sim ND´s work. It is not reproducable from my side. Sorry for that.

But Problem with Flaps as discribed, is still present.

**Comment by Alex1337Cool** [comment]
Can confirm version ccf27c4 is the most recent version as per the Installer on my side as well

**Comment by MisterChocker** [comment]
Yeah sorry, seems to be ccf27c4 is the most recent change on the fbw end (which happened in February, hence the confusion).
The dev version does not seem to have the issue.

**Comment by MisterChocker** [comment]
I am unable to reproduce the reported issue.

**Comment by MisterChocker** [comment]
Do you have the flaps extrend/retract on an axis?
